bpf/bpf_lxc.c

```diff
@@ -1042,7 +1042,11 @@ ipv4_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label, __u8 *reason,
 
 	l4_off = l3_off + ipv4_hdrlen(ip4);
 	csum_l4_offset_and_flags(tuple.nexthdr, &csum_off);
+#if !(defined IPV4_FRAGMENTS && defined HAVE_LRU_MAP_TYPE)
```

Comment:
nit: Throughout the datapath code, there is this double-check for `IPV4_FRAGMENTS` and `HAVE_LRU_MAP_TYPE`, but I think we can skip the second check and only define `IPV4_FRAGMENTS` from userspace if the LRU type is supported?

Reply:
Yes, I should probably do that. I didn't know how `IPV4_FRAGMENTS` would be passed from user space when I wrote the BPF part.

Reply:
Updated in new commit, but this required me to add a bpftool probe in pkg/datapath/linux/config/config.go.

Reply:
That should be fine, the probe should be cached after the first time.